packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -443,7 +443,8 @@ const _BaseButton: React.ForwardRefRenderFunction<BladeElementRef, BaseButtonPro
   );
 };
 
-const BaseButton = React.forwardRef(_BaseButton);
-BaseButton.displayName = 'BaseButton';
+const BaseButton = /*#__PURE__*/ Object.assign(React.forwardRef(_BaseButton), {
```

Comment:
We can also add the explanation as comment to one particular component (probably button)

Reply:
We might need a lint rule to enforce this, easy to miss out. How did you even figure this out lol 😸

Reply:
Btw is there a babel plugin that automatically does this transformation? Seems like a common issue libraries might be running into 🤔

Reply:
There is this babel plugin - https://github.com/mbrowne/babel-plugin-pure-static-props. I tried this first but it didn't work. I think it works when props are on react component and not when they are on refs. Even for react components it didn't work at lots of places for some reason. Couldn't find any other plugin either.

Reply:
> We might need a lint rule to enforce this, easy to miss out. 

Yes definitely! Exploring if it can be eslint rule or babel plugin.

> How did you even figure this out lol 😸

Lol it was tricky